fastlane/Fastfile

```diff
@@ -89,26 +93,67 @@ platform :ios do
   desc "Release pods"
   private_lane :release_pods do
     AmplifyPods.pods.each do |pod|
-      pod_path = "/Users/distiller/amplify-ios/#{pod[:spec]}"
+      if pod[:no_push]
+        UI.message "Skipping pushing pod #{pod[:spec]}"
+        next
+      end
 
-      UI.message("Pushing pod #{pod[:spec]}")
+      pod_path = "/Users/distiller/amplify-ios/#{pod[:spec]}"
```

Comment:
Extract "/Users/distiller" to a constant, or use the `HOME` env var?

Also, looking ahead to re-using this in the aws-sdk-ios project, consider extracting the project name to a constant.